src/ray/common/test_util.h

```diff
@@ -56,18 +61,63 @@ extern std::string REDIS_MODULE_LIBRARY_PATH;
 /// Ports of redis server.
 extern std::vector<int> REDIS_SERVER_PORTS;
 
-/// Test helper class, it will start redis server before the test runs,
-/// and stop redis server after the test is completed.
-class RedisServiceManagerForTest : public ::testing::Test {
+/// Path to object store executable binary.
+extern std::string STORE_EXEC_PATH;
+
+/// Path to gcs server executable binary.
+extern std::string GCS_SERVER_EXEC_PATH;
+
+/// Path to raylet executable binary.
+extern std::string RAYLET_EXEC_PATH;
+/// Path to mock worker executable binary. Required by raylet.
+extern std::string MOCK_WORKER_EXEC_PATH;
+/// Path to raylet monitor executable binary.
+extern std::string RAYLET_MONITOR_EXEC_PATH;
+
+//--------------------------------------------------------------------------------
+// COMPONENT MANAGEMENT CLASSES FOR TEST CASES
+//--------------------------------------------------------------------------------
+/// Test cases can use it to 1) start redis server(s), 2) stop redis server(s) and 3)
+/// flush all redis servers.
+class RedisServiceManagerForTest {
```

Comment:
I think it would be easier to put all the helper functions below into a single class, which would be easier to remember:)  e.g. TestSetupUtil

Reply:
All right. Merge them into a one single class would make it bigger but it's easy to use. Will do it.